test/fixtures/schema.sql

```diff
@@ -113,6 +113,14 @@ CREATE TABLE items2 (
     id bigserial primary key
 );
 
+CREATE TABLE items3 (
+    id bigserial primary key
+);
+
+CREATE TABLE persistance (
+    id bigint primary key
+);
+
```

Comment:
@wolfgangwalther Hmm.. is this table used? I don't see it on the tests..

Reply:
Ah, good catch. Left over from something I tried. Will remove.